schema.sql

```diff
@@ -34,4 +34,28 @@ ALTER TABLE animals ADD COLUMN owner_id INT REFERENCES owners(id);
 
 
 
+CREATE TABLE vets (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(250),
+    age INT,
+    date_of_graduation DATE
+);
```

Comment:
- [ ] please remember to set the `id` as primary key

Reply:
Ok let me fix it
Thanks @mikethreels 👋🏿